test/utils/SystemPropertiesHelperSpec.scala

```diff
@@ -74,7 +74,7 @@ class SystemPropertiesHelperSpec extends AnyWordSpecLike with Matchers with Opti
   "When getting a String system property which does exist the helper" should {
     "return the default value" in {
       val s = new StubSystemProperties
-      assert(s.getStringProperty("searchString", null) == "foundString")
```

Comment:
getStringProperty is not being used. Can we remove it?

Reply:
done